pkg/lwip/contrib/lwip.c

```diff
@@ -250,6 +255,12 @@ void lwip_bootstrap(void)
 #endif
     /* also allow for external interface definition */
     tcpip_init(NULL, NULL);
+#if IS_USED(MODULE_LWIP_DHCP_AUTO) && IS_USED(MODULE_NETDEV_TAP)
+    /* XXX: Hack to get DHCP with IPv4 with `netdev_tap`, as it does
+     * not emit a `NETDEV_EVENT_LINK_UP` event. Remove, once it does
+     * at an appropriate point. */
```

Comment:
maybe explicitly mentioning this PR number makes it easier to get some context on what *appropriate* refers to here.

Reply:
Done